ssp

```diff
@@ -4272,14 +4272,15 @@ sub check_for_empty_easyapache_profiles {
         },
         $dir
     );
+    if (@easyapache_templates) {
+        my $templates;
+        for my $template (@easyapache_templates) {
```

Comment:
Try replacing all of this with:
```
if (@easyapache_templates) {
    print_warn("EA3 Empty template(s) in $dir: ");
    print_warning( join(' ', @easyapache_templates) );
}
```

Reply:
This has been done.

Verified once more:
Testing:
perlcritic: ✅ 
perltidy: ✅
No longer seeing this when no empty templates are present.
Seeing the warning when empty templates are present.